src/main/java/com/woowacourse/zzinbros/user/web/controller/FriendController.java

```diff
@@ -1,41 +1,52 @@
 package com.woowacourse.zzinbros.user.web.controller;
 
-import com.woowacourse.zzinbros.user.domain.User;
 import com.woowacourse.zzinbros.user.dto.FriendRequestDto;
+import com.woowacourse.zzinbros.user.dto.UserResponseDto;
 import com.woowacourse.zzinbros.user.service.FriendService;
-import com.woowacourse.zzinbros.user.service.UserService;
 import com.woowacourse.zzinbros.user.web.support.SessionInfo;
 import com.woowacourse.zzinbros.user.web.support.UserSession;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Set;
 
 @Controller
 @RequestMapping("/friends")
 public class FriendController {
 
-    private UserService userService;
     private FriendService friendService;
 
-    public FriendController(UserService userService, FriendService friendService) {
-        this.userService = userService;
+    public FriendController(FriendService friendService) {
         this.friendService = friendService;
     }
 
+    @GetMapping
+    @ResponseBody
+    public ResponseEntity<Set<UserResponseDto>> getFriends(@SessionInfo UserSession userSession) {
+        final UserResponseDto loginUserDto = userSession.getDto();
+        Set<UserResponseDto> friends = friendService.findFriendsByUser(loginUserDto);
+        return new ResponseEntity<>(friends, HttpStatus.OK);
```

Comment:
```suggestion
        return ResponseEntity.ok(friends);
```

https://www.baeldung.com/java-constructors-vs-static-factory-methods